tests/Violation/Renderer/Htmloutput/HtmlOutputRendererTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace SensioLabs\DeprecationDetector\Tests\Violation\Renderer\HtmlOutput;
+
+use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
+use PHPUnit_Framework_TestCase;
+use Prophecy\Argument;
+use SensioLabs\DeprecationDetector\Violation\Renderer\Html\HtmlRenderer;
+
+class HtmlOutputRendererTest extends PHPUnit_Framework_TestCase
```

Comment:
The class name should be changed into `HtmlRendererTest` or `RendererTest`, depends on [this comment](https://github.com/sensiolabs-de/deprecation-detector/pull/58/files#r43491096)